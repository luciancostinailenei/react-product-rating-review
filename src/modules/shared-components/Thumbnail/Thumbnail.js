import React from 'react'

const Thumbnail = props => {
  return (
    <div className="Thumbnail">
      <img
        className={'thumbnail thumbnail--' + props.type}
        src={props.source}
        alt={props.alt}
      />
    </div>
  )
}

export default Thumbnail
