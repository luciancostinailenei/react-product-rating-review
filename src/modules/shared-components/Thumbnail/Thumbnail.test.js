import React from 'react'
import toJson, { createSerializer } from 'enzyme-to-json'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import './Thumbnail'
import Thumbnail from './Thumbnail'

configure({ adapter: new Adapter() })

describe('ProductReview', () => {
  it('matches snapshot', () => {
    const ThumbnailWrapper = shallow(<Thumbnail />)

    expect(toJson(ThumbnailWrapper)).toMatchSnapshot()
  })
})
