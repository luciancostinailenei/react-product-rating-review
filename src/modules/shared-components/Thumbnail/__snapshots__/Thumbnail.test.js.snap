// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductReview matches snapshot 1`] = `
<div
  className="Thumbnail"
>
  <img
    className="thumbnail thumbnail--undefined"
  />
</div>
`;
